### DISCUSSION TITLE
Is there a way to make electron desktop bundle size smaller

### DISCUSSION THREAD

**Discussion by af4007** [atom/atom#22264] [Q&A]
Currently we are evaluating **electron-9.4.0** for windows desktop app development, and we have package size constraints while shipping the app, and I can see electron simple 'Hello, World!' app takes around 115MB for package, is there any way to reduce it under 40MB(simple plain app, not installer), without compromising existing functionality?

**Comment by m1ga** [comment]
I think you should ask in an electron community forum: https://www.electronjs.org/community
